prelude/bootstrap.js

```diff
@@ -197,10 +197,10 @@ function copyFolderRecursiveSync(source, target) {
       if (fs.lstatSync(curSource).isDirectory()) {
         copyFolderRecursiveSync(curSource, targetFolder);
       } else {
-        fs.copyFileSync(
-          curSource,
-          path.join(targetFolder, path.basename(curSource))
-        );
+        const curTarget = path.join(targetFolder, path.basename(curSource));
+        if (!fs.existsSync(curTarget)) {
+          fs.copyFileSync(curSource, curTarget);
+        }
```

Comment:
The change is good, I would add a comment here explaining why we need this (maybe also add a reference to the issue solved)

Reply:
Sure, I will add this

Reply:
Done, [0269890](https://github.com/vercel/pkg/pull/1611/commits/02698905071de369f1a9a235eb142e0796387200)